scripts/version.js

```diff
@@ -0,0 +1,28 @@
+const fs = require('fs');
+const { execSync } = require('child_process');
+const { version: NEW_VERSION } = require('../package.json');
+
+async function main() {
```

Comment:
Oh this is cool!

Reply:
😊 Haha, one of the benefits of not having to worry about client environments is that I can use latest node features :P